doc/api/next_api_changes/behavior/22347-RQ.rst

```diff
@@ -0,0 +1,13 @@
+Correctly treat pan/zoom events of overlapping axes
```

Comment:
```suggestion
Correctly treat pan/zoom events of overlapping Axes
```

We've decided to captialize this, when referring to the Axes (class) as a concept so that this is not confused with the plural of "axis". See #18726. Should also be changed for all other "Axes" mentionings below.